File: test/java/com/marsrovers/location/PlateauFactoryTest.java
package com.marsrovers.location;


import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class PlateauFactoryTest {

    PlateauFactory factory = new PlateauFactory();

    @Test
    public void defaultShouldCreateA10by10Plateau() throws Exception {

        Plateau plateau = factory.createDefaultPlateau();

        assertEquals(10, plateau.horizontalBoundary);
        assertEquals(10, plateau.verticalBoundary);

    }
}

File: test/java/com/marsrovers/location/PlateauFactoryTest.kt
package com.marsrovers.location


import org.junit.Test

import org.junit.Assert.assertEquals

class PlateauFactoryTest {

    internal var factory = PlateauFactory()

    @Test
    @Throws(Exception::class)
    fun defaultShouldCreateA10by10Plateau() {

        val plateau = factory.createDefaultPlateau()

        assertEquals(10, plateau.horizontalBoundary.toLong())
        assertEquals(10, plateau.verticalBoundary.toLong())

    }
}
